man/cc-test-reporter-format-coverage.1.md

```diff
@@ -33,8 +33,7 @@ tests were run. Defaults to current working directory.
 
 ## COVERAGE_FILE
 
-Path where the coverage file is located. It requires the flag `--input-type`
-to be set.
+(Optional) Explicit path where the coverage file is located. If not passed, the path to the file is inferred based on standard some coverage tool locations. If specifying an explicit COVERAGE_FILE, an `--input-type` must also be specified.
```

Comment:
i think why i want to say is that doing this `cc-test-reporter-format-coverage some/path`  will cause `some/path` to not be taken into account, unless you pass `--input-type`. It won't error, it will just look for the hardcoded path. 

sorry that i didn't explain myself better before.